src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -382,7 +382,7 @@ const TransactionDetailsComponent = (props: Props) => {
       {transaction.spendTargets == null ? null : (
         <EdgeAnim enter={{ type: 'fadeInDown', distance: 60 }}>
           <CardUi4>
-            <RowUi4 rightButtonType="copy" title={lstrings.transaction_details_recipient_addresses} body={recipientsAddresses} />
+            <RowUi4 maximumHeight="large" rightButtonType="copy" title={lstrings.transaction_details_recipient_addresses} body={recipientsAddresses} />
```

Comment:
This fix doesn't work for a number of addresses. The legacy sm/med/large prop
from Tile I think we should deprecate.

I think a better fix for now is to give a complex child that properly handles
printing out the addresses on individual lines.